src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor.CoreComponents/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Remote.Razor.CoreComponents
+{
+    internal static class Program
+    {
+#pragma warning disable IDE0060 // Remove unused parameter
+        public static void Main(string[] args)
```

Comment:
Unfortunate to require an empty `static void Main`. I assume this is required because the project is an exe for some kind of package load reason?

Reply:
> Unfortunate to require an empty `static void Main`. I assume this is required because the project is an exe for some kind of package load reason?

Ya project's an exe to make it publish its entire dependency graph so we can then later absorb it into the VSIX.

> Not sure I understand ServiceHub and the package load tricks being used here enough to give you a meaningful approval, is there any documentation or articles I can brush up on those with?

From what I've seen no? @genlu has been the information champ in the area 😄

Reply:
I used the publish task of a console app as a way to capture all dependencies, might be better way to do it but I don't have time to figure that out yet.

Reply:
Fair enough, and I don't have a better suggestion.